### PR TITLE
a missing word inserted

### PR DESCRIPTION
**Pull request by ercan7**

### CODE REVIEW

**Comment by ercan7** [comment]
In the book, it says "whenever you plot a mean (or other summary) vs. group size, you’ll see that the variation decreases as the sample size increases". This is a result of the LLN because according to the LLN, as the sample increases the sample mean converges to the population mean and therefore the variation is gonna be less. On the other hand, the CLT is about the distribution of the sample means, not the variation in sample means.

File: data-transform.qmd [hunk 1]
```diff
@@ -11,7 +11,7 @@ status("complete")
 
 Visualisation is an important tool for generating insight, but it's rare that you get the data in exactly the right form you need for it.
 Often you'll need to create some new variables or summaries to see the most important patterns, or maybe you just want to rename the variables or reorder the observations to make the data a little easier to work with.
-You'll learn how to do all that (and more!) in this chapter, which will introduce you to data transformation using the **dplyr** package and a new dataset on flights that departed New York City in 2013.
+You'll learn how to do all that (and more!) in this chapter, which will introduce you to data transformation using the **dplyr** package and a new dataset on flights that departed from New York City in 2013.
```
Comment (hadley) [inline data-transform.qmd:14]: I don't think from is needed here.

File: data-transform.qmd [hunk 2]
```diff
@@ -699,9 +699,9 @@ ggplot(delays, aes(x = n, y = delay)) +
 ```
 
 Not surprisingly, there is much greater variation in the average delay when there are few flights for a given plane.
-The shape of this plot is very characteristic: whenever you plot a mean (or other summary) vs. group size, you'll see that the variation decreases as the sample size increases[^data-transform-4].
+The shape of this plot is very characteristic: whenever you plot a mean (or other summary statistics) vs. group size, you'll see that the variation decreases as the sample size increases[^data-transform-4].
```
Comment (hadley) [inline data-transform.qmd:702]: I think using summary here is fine.

File: data-transform.qmd [hunk 3]
```diff
@@ -699,9 +699,9 @@ ggplot(delays, aes(x = n, y = delay)) +
...
 
-[^data-transform-4]: \*cough\* the central limit theorem \*cough\*.
+[^data-transform-4]: \*cough\* the law of large numbers \*cough\*.
```
Comment (hadley) [inline data-transform.qmd:704]: Good catch!

File: data-transform.qmd (hunk 1 above)
Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:14]:
```suggestion
You'll learn how to do all that (and more!) in this chapter, which will introduce you to data transformation using the **dplyr** package and a new dataset on flights that departed New York City in 2013.
```

**Comment by mine-cetinkaya-rundel** [comment]
Thank you!

**Pull request merged**